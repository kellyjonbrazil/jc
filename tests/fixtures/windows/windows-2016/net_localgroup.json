{
  "account_origin": "\\\\WIN-SERVER16",
  "domain": null,
  "comment": null,
  "groups": [
    {
      "name": "Access Control Assistance Operators",
      "members": []
    },
    {
      "name": "Administrators",
      "members": []
    },
    {
      "name": "Backup Operators",
      "members": []
    },
    {
      "name": "Certificate Service DCOM Access",
      "members": []
    },
    {
      "name": "Cryptographic Operators",
      "members": []
    },
    {
      "name": "Distributed COM Users",
      "members": []
    },
    {
      "name": "Event Log Readers",
      "members": []
    },
    {
      "name": "Guests",
      "members": []
    },
    {
      "name": "Hyper-V Administrators",
      "members": []
    },
    {
      "name": "IIS_IUSRS",
      "members": []
    },
    {
      "name": "Network Configuration Operators",
      "members": []
    },
    {
      "name": "Performance Log Users",
      "members": []
    },
    {
      "name": "Performance Monitor Users",
      "members": []
    },
    {
      "name": "Power Users",
      "members": []
    },
    {
      "name": "Print Operators",
      "members": []
    },
    {
      "name": "RDS Endpoint Servers",
      "members": []
    },
    {
      "name": "RDS Management Servers",
      "members": []
    },
    {
      "name": "RDS Remote Access Servers",
      "members": []
    },
    {
      "name": "Remote Desktop Users",
      "members": []
    },
    {
      "name": "Remote Management Users",
      "members": []
    },
    {
      "name": "Replicator",
      "members": []
    },
    {
      "name": "Storage Replica Administrators",
      "members": []
    },
    {
      "name": "System Managed Accounts Group",
      "members": []
    },
    {
      "name": "Users",
      "members": []
    }
  ]
}
