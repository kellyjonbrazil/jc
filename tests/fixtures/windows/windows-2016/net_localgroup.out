Aliases for \\WIN-SERVER16

-------------------------------------------------------------------------------
*Access Control Assistance Operators
*Administrators
*Backup Operators
*Certificate Service DCOM Access
*Cryptographic Operators
*Distributed COM Users
*Event Log Readers
*Guests
*Hyper-V Administrators
*IIS_IUSRS
*Network Configuration Operators
*Performance Log Users
*Performance Monitor Users
*Power Users
*Print Operators
*RDS Endpoint Servers
*RDS Management Servers
*RDS Remote Access Servers
*Remote Desktop Users
*Remote Management Users
*Replicator
*Storage Replica Administrators
*System Managed Accounts Group
*Users
The command completed successfully.